src/labels/CustomLabels.labels

```diff
@@ -4628,6 +4628,22 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>stgCreate</shortDescription>
         <value>Create</value>
     </labels>
+    <labels>
+        <fullName>stgCRLPHouseholdAccountError</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>stgCRLPHouseholdAccountError</shortDescription>
+        <value>You must be using the Household Account Model in order to use Customizable Rollups.</value>
+    </labels>
+    <labels>
+        <fullName>stgCRLPNonAdminError</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>false</protected>
```

Comment:
I believe all custom labels should be set with protected = true. @jjbennett Can you confirm that?

Reply:
Confirmed - essentially it makes the class public to our namespace but private to others and setting it to "false" would be similar to making it global.